web/react/tslint.json

```diff
@@ -0,0 +1,68 @@
+{
+  "extends": ["tslint-react", "tslint-no-circular-imports"],
+  "defaultSeverity": "warning",
```

Comment:
Perhaps you want to remove this line?
Not sure if default severity is "error". I guess the linter should make fail CI and probably this line will make it pass with just warnings. Although, also I'm not sure if this is intentional for now.

Reply:
done .. I took that from kiali-ui but you're right we can be stricter

Reply:
In kiali-ui, Travis uses https://github.com/kiali/kiali-ui/blob/master/tslint.prod.json... That way, while developing, `yarn start` works with warnings, but Travis fails with errors :)

Reply:
ah ok, fair enough!